cicd/pyinstaller/requirment.txt

```diff
@@ -25,3 +25,4 @@ cryptography==2.8
 python-crontab==2.5.1
 confluent-kafka==1.5.0
 netifaces==0.10.9
+tzlocal==2.1
```

Comment:
can't we achieve it without adding third party?

Reply:
I could not find way to get local timezone.

Reply:
removed tzlocal module.